README.md

```diff
@@ -1,251 +1,220 @@
-# stargate-mongoose
+# stargate-mongoose ![ci-tests](https://github.com/stargate/stargate-mongoose/actions/workflows/ci-tests.yml/badge.svg)
 
-![ci-tests](https://github.com/stargate/stargate-mongoose/actions/workflows/ci-tests.yml/badge.svg)
-
-`stargate-mongoose` is a mongoose driver for [JSON API Server](https://github.com/stargate/jsonapi)/[Astra DB](https://astra.datastax.com).
-
-## Table of contents
+`stargate-mongoose` is a mongoose driver for [JSON API Server](https://github.com/stargate/jsonapi) which runs on top of Apache Cassandra / DataStax Enterprise.
```

Comment:
nit: I don't like calling it a "Server". I'd just say JSON API.

Reply:
Also, should we capitalize Mongoose in text (not commands of course)

Reply:
Done